src/basedtyping/__init__.py

```diff
@@ -473,15 +466,10 @@ def Untyped(  # noqa: N802
         This is more specialized than ``Any`` and can help with gradually typing modules.
         """
         raise TypeError(f"{self} is not subscriptable")
-
 else:
-    Untyped: Final = _BasedSpecialForm(
-        "Untyped",
-        doc=(
-            "Special type indicating that something isn't typed.\nThis is more"
-            " specialized than ``Any`` and can help with gradually typing modules."
-        ),
-    )
+    # We pretend that it's an alias to Any so that it's slightly more compatible with
+    #  other tools
+    Untyped: TypeAlias = Any  # type: ignore[no-any-explicit]
```

Comment:
can we just get rid of `Untyped` its completely useless and is just confusing/misleading

Reply:
ahhhhhhhhhhhh, no